docs/SPEC.md

```diff
@@ -1464,7 +1464,7 @@ AggregateWindow has the following properties:
 | ----        | ----                                            | -----------                                                                                                           |
 | every       | duration                                        | Every specifies the window size to aggregate.                                                                         |
 | fn          | (tables: <-stream, columns: []string) -> stream | Fn specifies the aggregate operation to perform. Any of the functions in this Aggregate section can be provided.      |
-| columns     | []string                                        | Columns specifies a list of columns to aggregate. Defaults to ["_value"].                                             |
```

Comment:
Nit: in the line above, we say:

> Any of the functions in this Aggregate section can be provided.

This is not true. Covariance is here, but cannot be provided. It could be changed to:

```
Any aggregate function that accepts a `column` property can be provided.
```

Reply:
good point.  will fix.